### DISCUSSION TITLE
mqtt-client  can not compile on cc2538dk?

### DISCUSSION THREAD

**Discussion by tlzfighting** [contiki-ng/contiki-ng#2378] [Q&A] [answered]
This problem occurred when I used the following compile command
**### contiki-ng/examples/mqtt-client$   sudo make TARGET=cc2538dk build/cc2538dk/mqtt-client.hex**

CC        ../../arch/cpu/cc2538/cc2538.lds
  LD        build/cc2538dk/mqtt-client.elf
/usr/local/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: build/cc2538dk/obj/builtin-sensors.o:(.bss.tmp_buf+0x0): multiple definition of `tmp_buf'; build/cc2538dk/obj/als-extend.o:(.bss.tmp_buf+0x0): first defined here
collect2: error: ld returned 1 exit status
make: *** [../../arch/cpu/arm/cortex-m/Makefile.customrules-cortex-m:11: build/cc2538dk/mqtt-client.elf] Error 1

How to compile successfully, look forward to your advice, thank you!

**Comment by Mbaron12** [comment, marked as answer]
I have got it !
